internal/app/connectconformance/config.go

```diff
@@ -0,0 +1,395 @@
+// Copyright 2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connectconformance
```

Comment:
The reasoning was just so that if you are invoking a command-line program, running a program named `conformance` is a bit ambiguous. These conformance tests will also test gRPC and gRPC-Web conformance, so I don't think we'd have different programs for each. It was more about so that the set of programs in someone's `~/go/bin` has somewhat self-describing names.

I can rename it if you'd prefer. I wasn't sure of a better name to call it.